pkgs/development/python-modules/drf-standardized-errors/default.nix

```
@@ -0,0 +1,65 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  fetchpatch,
+  flit-core,
+  django,
+  djangorestframework,
+  drf-spectacular,
+  inflection,
+  pytestCheckHook,
+  pytest-django,
+  django-filter,
+}:
+
+buildPythonPackage rec {
+  pname = "drf-standardized-errors";
+  version = "0.14.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ghazi-git";
+    repo = pname;
```

```suggestion
    repo = "drf-standardized-errors";
```
we don't want to do that pattern anymore